docs/user/adding-new-metrics.md

```diff
@@ -1,8 +1,9 @@
 # Adding new metrics
 
-All metrics that your project collects must be defined in a `metrics.yaml` file.
-This file should be at the root of the application or library module (the same directory as the `build.gradle` file you updated).
-The format of that file is documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
+All metrics that your project collects must be defined in a `metrics.yaml`
```

Comment:
Reformatting just to fit max-line-length?
(I prefer [semantic linebreaks](https://sembr.org/), which make following PRs to the same text more readable most of the time)

Reply:
Sure -- I guess we should be consistent on this though (and part of the consistency issue is from *not* doing this in glean-ac).  I'm fine with semantic linebreaks, though, and might as well take care of making that consistent now.

Reply:
Ok. I'm fine either way.